pyirf/interpolation/component_estimators.py

```diff
@@ -0,0 +1,673 @@
+            Grid of effective area. Dimensions but the first can in principle be freely
+            chosen. Class is AEFF2D compatible, which would require
+            shape=(n_points, n_energy_bins, n_fov_offset_bins).
+        interpolator_cls:
+            pyirf interpolator class, defaults to GridDataInterpolator.
+        interpolator_kwargs: dict
+            Dict of all kwargs that are passed to the interpolator, defaults to
+            None which is the same as passing an empty dict.
+        extrapolator_cls:
+            pyirf extrapolator class. Can be and defaults to ``None``,
+            which raises an error if a target_point is outside the grid
+            and extrapolation would be needed.
+        extrapolator_kwargs: dict
+            Dict of all kwargs that are passed to the extrapolator, defaults to
+            None which is the same as passing an empty dict.
+        min_effective_area: astropy.units.Quantity[area]
+            Minimum value of effective area to be considered for interpolation. Values
+            lower then this value are set to this value. Defaults to 1 m**2.
+
+
+        Note
+        ----
+            Also calls __init__ of pyirf.component_estimators.BaseComponentEstimator
+            and ParametrizedEstimator
+        """
+
+        # get rid of units
+        effective_area = effective_area.to_value(u.m**2)
+        min_effective_area = min_effective_area.to_value(u.m**2)
+
+        self.min_effective_area = min_effective_area
+
+        # remove zeros and log it
+        effective_area[
+            effective_area < self.min_effective_area
+        ] = self.min_effective_area
+        effective_area = np.log(effective_area)
+
+        super().__init__(
+            grid_points=grid_points,
+            params=effective_area,
+            interpolator_cls=interpolator_cls,
+            interpolator_kwargs=interpolator_kwargs,
+            extrapolator_cls=extrapolator_cls,
+            extrapolator_kwargs=extrapolator_kwargs,
+        )
+
+    def __call__(self, target_point):
+        """
+        Estimating effective area at target_point, inter-/extrapolates as needed and
+        specified in __init__.
+
+        Parameters
+        ----------
+        target_point: np.ndarray, shape=(1, n_dims)
+            Target for inter-/extrapolation
+
+        Returns
+        -------
+        aeff_interp: np.ndarray of (astropy.units.m)**2, shape=(n_points, ...)
+            Interpolated Effective area array with same shape as input
+            effective areas. For AEFF2D of shape (n_energy_bins, n_fov_offset_bins)
+
+        """
+
+        aeff_interp = super().__call__(target_point)
+
+        # exp it and set to zero too low values
+        aeff_interp = np.exp(aeff_interp)
+        # 1.1 to correct for numerical uncertainty and interpolation
+        aeff_interp[aeff_interp < self.min_effective_area * 1.1] = 0
```

Comment:
10 % is rather large... and it's a bit surprising. `min_` should be `min_` not 10 % lower than min...